src/register/mod.rs

```diff
@@ -61,13 +65,12 @@ pub mod mimpid;
 pub mod mvendorid;
 
 // Machine Trap Setup
-pub mod medeleg;
-pub mod mideleg;
```

Comment:
Why you removed these modules?

Reply:
I'm not sure how they got removed, my github fork had them there but my local repo had them as still //TODO. FIxed